tests/migration_test.go

```diff
@@ -337,6 +337,35 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 		time.Sleep(15 * time.Second)
 	}
 
+	getLibvirtdPid := func(pod *k8sv1.Pod) string {
+		stdout, _, err := tests.ExecuteCommandOnPodV2(virtClient, pod, "compute",
```

Comment:
> > Did you check for pkill?
> 
> pkill might have worked too. However, I wanted the test to prove that the pid cycled, so using `ps` to get the pid, `kill` to signal the pid, and then ps again to find the new pid seemed to work. Since I already know the pid in that example, i just used kill.
> 
> > Was it intentional to replace a graceful kill with a.. non-graceful one? (-9)
> 
> my only thought here was predictability. I know with -9 that the pid has to go away and there's no timing to account for.

I think that you should use pgrep to get the pid. It will keep the initial functionality and eliminate the need for this loop that parses the output of ps -x

Reply:
I wanted the test to prove that the pid cycled, so using ps to get the pid, kill to signal the pid, and then ps again to find the new pid seemed to work. Since I already know the pid in that example, i just used kill.